Header header
string id
float64 signal_strength
float64 noise
string essid
float64 channel
